srcvideos/Makefile

```diff
@@ -0,0 +1,118 @@
+PY?=python3
+PELICAN?=pelican
+PELICANOPTS=
+
+BASEDIR=$(CURDIR)
+INPUTDIR=$(BASEDIR)/content
+OUTPUTDIR=$(BASEDIR)/../videos
+DATADIR=$(BASEDIR)/data
+CONFFILE=$(BASEDIR)/pelicanconf.py
+PUBLISHCONF=$(BASEDIR)/publishconf.py
+
+GITHUB_PAGES_REPO=git@github.com:andife/onnxvideo.github.io.git
```

Comment:
what should this be for ONNX site?

Reply:
it depends... we can also get rid of it.... because I don't expect the dynamics or multitude of events as with pyvideo. If it mainly relates to the Community Days, it may remain manageable.

originally it was thought on then to refer to https://github.com/onnx/onnx.github.io/tree/main/srcvideos after merge with current proposed structure

Reply:
> we generally avoid including files that cannot be rendered in GitHub. Can the ODS format files be replaced with CSVs or JSON?
> 
> Also, since the individual JSONs are generated from it, can we run that as part of the CI pipeline instead of adding them separately?

I fully understand that a format that is not readable in the brower / at github is not particularly good at the repo.
[Would refrain from checking it in...]
 
Converting this "large" file to json or csv is not really an option in my opinion,
* because if it is an json how to I edit it with a nice gui like libreoffice and add maybe multiple rows with the some content to the file?
I did not really find suitable software for that. But if you or someone else have some hints?
* if it's a csv... I could it probably work...

In general I would have stayed with the single files, because this would allow to change single json-files and add information without changing the whole pipeline. If we go in the direction of the pipeline, the quality of the scripts would have to be changed there as well...).